// @expo-google-fonts/poppins/index.js
// This file is generated by @expo-google-fonts/generator
// If you want to modify it, go to packages/generator/templates

export * from './useFonts';
export { default as __metadata__ } from './metadata.json';
export const Poppins_100Thin = require('./100Thin/Poppins_100Thin.ttf');
export const Poppins_100Thin_Italic = require('./100Thin_Italic/Poppins_100Thin_Italic.ttf');
export const Poppins_200ExtraLight = require('./200ExtraLight/Poppins_200ExtraLight.ttf');
export const Poppins_200ExtraLight_Italic = require('./200ExtraLight_Italic/Poppins_200ExtraLight_Italic.ttf');
export const Poppins_300Light = require('./300Light/Poppins_300Light.ttf');
export const Poppins_300Light_Italic = require('./300Light_Italic/Poppins_300Light_Italic.ttf');
export const Poppins_400Regular = require('./400Regular/Poppins_400Regular.ttf');
export const Poppins_400Regular_Italic = require('./400Regular_Italic/Poppins_400Regular_Italic.ttf');
export const Poppins_500Medium = require('./500Medium/Poppins_500Medium.ttf');
export const Poppins_500Medium_Italic = require('./500Medium_Italic/Poppins_500Medium_Italic.ttf');
export const Poppins_600SemiBold = require('./600SemiBold/Poppins_600SemiBold.ttf');
export const Poppins_600SemiBold_Italic = require('./600SemiBold_Italic/Poppins_600SemiBold_Italic.ttf');
export const Poppins_700Bold = require('./700Bold/Poppins_700Bold.ttf');
export const Poppins_700Bold_Italic = require('./700Bold_Italic/Poppins_700Bold_Italic.ttf');
export const Poppins_800ExtraBold = require('./800ExtraBold/Poppins_800ExtraBold.ttf');
export const Poppins_800ExtraBold_Italic = require('./800ExtraBold_Italic/Poppins_800ExtraBold_Italic.ttf');
export const Poppins_900Black = require('./900Black/Poppins_900Black.ttf');
export const Poppins_900Black_Italic = require('./900Black_Italic/Poppins_900Black_Italic.ttf');
